pkg/morph/client/constructor.go

```diff
@@ -99,6 +102,13 @@ func New(key *keys.PrivateKey, endpoint string, opts ...Option) (*Client, error)
 	}, nil
 }
 
+func initClientCache() cache {
+	lru, _ := lru.New(100) // panics only if size isn't negative
```

Comment:
"panics", "isn't negative"?

Reply:
still did not get why the "panics" word was used

Reply:
Oh, got that. Fixed.